addons/account/models/account_journal_dashboard.py

```diff
@@ -131,7 +131,7 @@ def _query_has_sequence_holes(self):
               FROM account_move move
               JOIN res_company company ON company.id = move.company_id
              WHERE move.journal_id = ANY(%(journal_ids)s)
-               AND (move.state = 'posted' OR (move.state = 'draft' AND move.name != '/'))
+               AND (move.state = 'posted' OR (move.state = 'draft' AND move.sequence_number != 0 AND move.name = '/'))
```

Comment:
How is it possible to have `sequence_number != 0` with `name = '/'`?

Reply:
You're right, I did that to have an impossible condition (I mean not consciously). We can remove the whole part with the `move.state = 'draft'`. It's no longer necessary.